Wiki/Programs/PokemonSwSh/EggFetcherMultiple.md

```diff
@@ -0,0 +1,63 @@
+# Egg Fetcher Multiple
+
+## Program Description
+
+Fetch eggs into your boxes without hatching them.
+
+This program is used with [EggHatcher](EggHatcher.md) to fetch and hatch eggs. Thus you first fetch a bunch of eggs with EggFetcher, then you hatch them with EggHatcher.
+
+Unlike the other egg fetcher, it will fetch a chosen number of eggs before switching one parent and trying again.
```

Comment:
Image Kudasai? :)

Reply:
I used the egg fetcher 2 image because it is exactly the same location to start the program